packages/adapter/src/lib/adapter/adapter.ts

```diff
@@ -3428,6 +3428,60 @@ export class AdapterClass extends EventEmitter {
         }
     }
 
+    // external signature
+    objectExists(id: string, options?: Record<string, any>): Promise<boolean> | void;
+
+    /**
+     * Checks if an object exists to the given id, id will be fixed first
+     *
+     * @param id id of the object
+     * @param options optional user context
+     */
+    objectExists(id: unknown, options: unknown): Promise<boolean> | void {
+        if (!adapterObjects) {
+            this._logger.info(`${this.namespaceLog} objectExists not processed because Objects database not connected`);
+            return;
+        }
+
+        Utils.assertString(id, 'id');
+        if (options !== undefined) {
+            Utils.assertObject(options, 'options');
+        }
+
+        id = this._utils.fixId(id);
+
+        this._utils.validateId(id, false, null);
+
+        return adapterObjects.objectExists(id, options);
+    }
+
+    // external signature
+    foreignObjectExists(id: string, options?: Record<string, any>): Promise<boolean> | void;
+
+    /**
+     * Checks if an object exists to the given id
+     *
+     * @param id id of the object
+     * @param options optional user context
+     */
+    foreignObjectExists(id: unknown, options: unknown): Promise<boolean> | void {
+        if (!adapterObjects) {
+            this._logger.info(
+                `${this.namespaceLog} foreignObjectExists not processed because Objects database not connected`
+            );
+            return;
+        }
+
+        Utils.assertString(id, 'id');
+        if (options !== undefined) {
```

Comment:
und null? :-)

Reply:
it is last arg, so no need to pass anything but we can allow null